data_model/Cargo.toml

```diff
@@ -55,6 +55,7 @@ base64 = { version = "0.13.1", default-features = false, features = ["alloc"]}
 [dev-dependencies]
 iroha_core = { path = "../core", version = "=2.0.0-pre-rc.13" }
 iroha_client = { path = "../client", version = "=2.0.0-pre-rc.13" }
+iroha_genesis = { path = "../genesis", version = "=2.0.0-pre-rc.13" }
 iroha = { path = "../cli" }
 
 test_network = { path = "../core/test_network", version = "=2.0.0-pre-rc.13" }
```

Comment:
can we remove this dependency? Maybe these tests should actually reside in the `client/tests/integration`?

Reply:
I think only `find_rate_and_make_exchange_isi_should_succeed` should be moved to `iroha_client`